webapp/src/gitjson.tsx

```diff
@@ -18,6 +18,9 @@ import * as pxtblockly from "../../pxtblocks";
 import IProjectView = pxt.editor.IProjectView;
 import UserInfo = pxt.editor.UserInfo;
 
+import { Accordion, AccordionItem, AccordionHeader, AccordionPanel } from "../../react-common/components/controls/Accordion/Accordion";
```

Comment:
As a design, it might be nice to scope the child components to the Accordion parent type, e.g.:
```jsx
    <Accordion>
        <Accordion.Item>
            <Accordion.Header>
                ...
            </Accordion.Header>
            <Accordion.Panel>
                ...
            </Accordion.Panel>
    </Accordion>
```

Reply:
Example here: https://github.com/microsoft/pxt/blob/master/teachertool/src/components/Toolbar.tsx

Reply:
done!